src/ParaTest/Runners/PHPUnit/Worker.php

```diff
@@ -32,7 +32,12 @@ public function start($wrapperBinary, $token = 1, $uniqueToken = null)
         if ($uniqueToken) {
             $bin .= "UNIQUE_TEST_TOKEN=$uniqueToken ";
         }
-        $bin .= "exec \"$wrapperBinary\"";
+        if (defined('HHVM_VERSION')) {
```

Comment:
no, this is the recommended way http://hhvm.com/blog/2393/hhvm-2-3-0-and-travis-ci